spatialos-sdk/src/worker/connection.rs

```diff
@@ -217,34 +217,6 @@ impl WorkerConnection {
         WorkerConnectionFuture::new(future_ptr)
     }
 
-    pub fn connect_locator_and_queue_async(
-        locator: &Locator,
-        deployment_name: &str,
-        params: &ConnectionParameters,
-        callback: QueueStatusCallback,
-    ) -> WorkerConnectionFuture {
-        let deployment_name_cstr = CString::new(deployment_name).unwrap();
-        let connection_params = params.to_worker_sdk();
-
-        let callback = Box::new(callback);
-        let callback_ptr = Box::into_raw(callback) as *mut ::std::os::raw::c_void;
-
-        unsafe {
-            let ptr = Worker_Locator_ConnectAndQueueAsync(
-                locator.locator,
-                deployment_name_cstr.as_ptr(),
-                &connection_params,
-                callback_ptr,
-                Some(queue_status_callback_handler),
```

Comment:
This `queue_status_callback_handler` can be removed as well now I think

Reply:
I think I did delete it?